app/src/main/java/com/github/polypoly/app/map/LocalizationRepository.kt

```diff
@@ -0,0 +1,94 @@
+package com.github.polypoly.app.map
+
+import com.github.polypoly.app.game.Localization
+import com.github.polypoly.app.game.LocalizationLevel
+import com.github.polypoly.app.game.Zone
+import org.osmdroid.util.GeoPoint
+
+/**
+ * Repository for providing the zones and localizations on the map.
+ */
+object LocalizationRepository {
+
+    /**
+     * List of localizations in the EPFL campus area.
+     */
+    private val localizations = listOf(
```

Comment:
If I understand correctly, this class is used to represent the buildings in general in a game. I think that it could be good to separate this class in two classes because otherwise it could complicate things because here we mix two different concepts:
- The buildings available in general in all the games. That is to say an exhaustive list of the buildings of the EPFL that will never change. 
- The current state of the buildings in the game (the current level, how much is the tax now, etc)

I propose two classes to represent these two concepts:
- `object LocalisationRepository` contains all the Localization possible in the game, with their name, their zone, their position, and evently a price table but I think it's better to calculate it according to the zone.
- `class LocalisationState` contains a list with the references of localisations of `LocalisationRepository`, their current owners, their current levels, and a method to calculate the tax based on the price and the level of the localisations. We can give an instance of this class as an attribute of a game.

Reply:
As you are pointing out, it is indeed better for these two concepts to be separated, I will simply remove the currentLevel attribute as it seems to be the only one that would better fit LocalizationState, the rest of the current attributes seem to be intrinsically linked to the localization itself and should not change during the game.
I'm down to work on LocalizationState next week though !